auth.go

```diff
@@ -516,7 +549,44 @@ func verifyGroupIDMathch(linkID *big.Int, groupID int, requestID uint32, groupID
 	return nil
 }
 
+// VerifyToken performs verification of jws/jwz token using the registered packers in package manager
+func (v *Verifier) VerifyToken(
+	token string,
+	opts ...pubsignals.VerifyOpt,
+) (*protocol.AuthorizationResponseMessage, error) {
+
+	var unpackOpts []iden3comm.PackerParams
+	cfg := pubsignals.VerifyConfig{}
+	for _, o := range opts {
+		o(&cfg)
+	}
+	if cfg.AcceptedProofGenerationDelay > 0 {
+		authV2Set, err := v.verificationKeyLoader.Load(circuits.AuthV2CircuitID)
+		if err != nil {
+			return nil, fmt.Errorf("failed upload circuits files: %w", err)
+		}
+		unpackOpts = append(unpackOpts, packers.NewZKPPUnpackerParams(authV2Set, v.ethResolvers, cfg.AcceptedProofGenerationDelay))
```

Comment:
See previous comment in  https://github.com/iden3/go-iden3-auth/pull/78/files#diff-1f48700829d7e0cdba7f3729d6a7c1c5df3aa11f2aeeed583c7e1e39aa7b66d1R152

 Here I'm passing authV2Set, and ethResolvers to be used inside Unpack function.